{"name": "test_login[case_data5]", "status": "passed", "attachments": [{"name": "log", "source": "019b475e-98b7-419e-957d-45c018e9d346-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "case_data", "value": "{'case_id': 6, 'case_title': '登陆失败-用户名、密码为空', 'case_data': {'userName': '', 'pwd': ''}, 'expected': {'code': '1001', 'msg': '用户名或密码错误！'}}"}], "start": 1645359564169, "stop": 1645359564284, "uuid": "d9b335b3-2bbf-45e4-8578-eabe1746f300", "historyId": "e7864f3f244908836f3b075a995c89f1", "testCaseId": "c54ba4c514553f220eeed2f7e798b554", "fullName": "testcase.test_login.TestLogin#test_login", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-7R8S57O"}, {"name": "thread", "value": "19860-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_login"}]}